### DISCUSSION TITLE
Can i set more than one SUPABASE_PUBLIC_URL?

### DISCUSSION THREAD

**Discussion by Steccah** [supabase/supabase#12652] [Questions]
I have a machine on my network that hosts supabase, i want to be able to connect to it from my network (setting SUPABASE_PUBLIC_URL with the ip in the local network) and to connect from outside my local network (setting SUPABASE_PUBLIC_URL with the network ip).
Is it possible to do it? Did i misconfigure somwthing?

**Comment by cristian-milea** [comment]
Maybe you can achieve this with a local DNS server in your home network. 
If anybody connected to your home network, goes to "mysupabase.steccah.com" from inside the home network, resolve to 192.168.0.xxx
